packages/backend/src/models/CatalogModel/CatalogModel.ts

```diff
@@ -216,6 +242,15 @@ export class CatalogModel {
             );
         }
 
+        catalogItemsQuery = catalogItemsQuery.groupBy(
+            `${CatalogTableName}.name`,
+            `${CatalogTableName}.description`,
+            `${CatalogTableName}.type`,
+            `${CachedExploreTableName}.explore`,
+            `${CatalogTableName}.required_attributes`,
+            `${CatalogTableName}.chart_usage`,
```

Comment:
```   `${CatalogTableName}.search_vector`,```

I think this is necessary because I just added a tag to a metric and search the catalog on the UI and it was throwing 500s